index.html

```diff
@@ -208,18 +215,20 @@ <h3 class="socials-header">LET'S CONNECT</h3>
             </section>
         <section class="contact-cont">
             <section class="contact-sec">
-                <h3>Contact me</h3>
-                <p>
-                    If you have an application you are interested in developing,
-                    a feature that you need built or a project that needs coding.
-                    I’d love to help with it 
-                </p>
-                <form action="https://formspree.io/f/mjvznweg" method="post">
-                    <input name="user-name" type="text" placeholder="Please enter your name" maxlength="30" required>
-                    <input name="user-email" type="email" placeholder="johndoe@gmail.com" required>
-                    <textarea name="user-msg" id="user-msg" cols="30" rows="10" maxlength="500" placeholder="write your message here" required></textarea>
-                    <button type="submit"class="btn">See project</button>
-                </form>
+                <div>
+                    <h3>Contact me</h3>
+                    <p>
+                        If you have an application you are interested in developing,
+                        a feature that you need built or a project that needs coding.
+                        I’d love to help with it 
+                    </p>
+                    <form action="https://formspree.io/f/mjvznweg" method="post">
+                        <input name="user-name" type="text" placeholder="Please enter your name" maxlength="30" required>
+                        <input name="user-email" type="email" placeholder="johndoe@gmail.com" required>
+                        <textarea name="user-msg" id="user-msg" cols="30" rows="10" maxlength="500" placeholder="write your message here" required></textarea>
+                        <button type="submit"class="btn">See project</button>
```

Comment:
- [ ] The text here should be `Get in Touch` not `See Project`. Kindly fix it just as the previous reviewer recommended.